cupy/math/misc.py

```diff
@@ -1,10 +1,14 @@
+import contextlib
+
 import cupy
 import cupyx
 import cupyx.scipy.fft
 
 from cupy import core
 from cupy.core import _routines_math as _math
 from cupy.core import fusion
+from cupy.cuda import cufft
+from cupy.fft.fft import _output_dtype
```

Comment:
Importing in this way is unavoidable because helper functions are not exposed to `cupy/fft/__init__.py`.